atomix/cluster/src/main/java/io/atomix/cluster/messaging/impl/NettyMessagingService.java

```diff
@@ -462,12 +477,15 @@ private void initEventLoopGroup() {
       return;
     }
 
+    // we need these to close them on stop
```

Comment:
Unfortunately I don't know how to add comments to lines which were not changed.  

My question is: Do we also need to add/remove the future for the local connection case above?

Reply:
No because they are executed anyway directly.